release-team/role-handbooks/enhancements/README.md

```diff
@@ -37,17 +34,17 @@ An Enhancements Lead holds the following responsibilities:
 - MUST have served on the Release Team in a previous capacity, ideally as an Enhancements Shadow
 - MUST be a member of the [Release Team Google Group][rt-group]
 - MUST be a member of the [SIG Release Google Group][sig-release-group]
-- MUST be a member of the [SIG PM Google Group][sig-pm-group]
+- MUST be a member of the [SIG Architecture Google Group][sig-arch-group]
 
 Helpful characteristics of an Enhancements Lead include:
 - experience with the Kubernetes community, code layout, ecosystem projects, organizational norms, governance, SIG structure, architecture, and release process
 - product / project / program management experience
 - release management experience
 
 Approximate Time Commitments
-Beginning of the cycle through enhancement freeze: 3-6 hours a week fluctuating based on how many SIG meetings need to be attended
-Enhancement Freeze through Code Freeze: 4-7 hours a week
-Code Freeze through Release Day: 1-4 hours a week
+- Beginning of the cycle through enhancement freeze: 3-6 hours a week fluctuating based on how many SIG meetings need to be attended
```

Comment:
Feels like 3-6 is a little low during prime time?

Reply:
I agree with that assessment. It's usually more than those many hours.

I feel 6-10hours could be a good ballpark. What do you think?